autonomi/src/client/data_types/chunk.rs

```diff
@@ -140,6 +145,115 @@ impl Client {
         }
     }
 
+    /// Manually upload a chunk to the network.
+    /// It is recommended to use the [`Client::data_put`] method instead to upload data.
```

Comment:
Just a nit, but if this comment line is meant as a new paragraph, insert an empty doc line (`///`) between them. Otherwise rustdoc will concatenate these sentences as a single line.

Reply:
Yeah it was just for our readability it doesn't have to be 2 different lines